x-pack/plugins/enterprise_search/public/applications/workplace_search/components/layout/nav.tsx

```diff
@@ -22,8 +22,9 @@ import {
 } from '../../routes';
 
 export const WorkplaceSearchNav: React.FC = () => {
-  const { enterpriseSearchUrl } = useContext(KibanaContext) as IKibanaContext;
-  const legacyUrl = (path: string) => `${enterpriseSearchUrl}/ws#${path}`;
```

Comment:
@scottybollinger FYI that some of these links have stopped working without the `#` - let me know if you'd like me to add the `#` as part of `externalUrl.getWorkplaceSearchUrl` if you'd rather update (for example) `ROLE_MAPPINGS_PATH` to have a leading `#` for now

Reply:
Hmm, disregard this - I fixed the broken ones manually in this https://github.com/elastic/kibana/pull/75093/commits/dfe1b5dd64100065bc7786f03f2caa96cb05647c